addons/web_editor/static/src/js/wysiwyg_snippets/snippets.editor.js

```diff
@@ -651,6 +651,7 @@ var SnippetsMenu = Widget.extend({
 
         core.bus.on('deactivate_snippet', this, this._onDeactivateSnippet);
         core.bus.on('snippet_editor_clean_for_save', this, this._onCleanForSaveDemand);
+        core.bus.on('toggle_mega_menu_snippets', this, this._toggleMegaMenuSnippets);
```

Comment:
Here is my latest detailed explanation: https://github.com/odoo/odoo/pull/31852#discussion_r281985023

Point 1: the code which toggles the menu should be in `SnippetsMenu`... yes but as always not in web_editor... mega menus are a website feature, web_editor should not know about their existence. How many times have I explained you this... Extend the snippets menu features to handle this case... in the **website** app.

Point 2: why did you use a bus to achieve that ? Can't you just do `this.snippetsMenu.toggleMegaMenuSnippets` in the `WysiwigMultizone` code ?